api/openapi-spec/swagger.json

```diff
@@ -54887,7 +54887,7 @@
     ]
    },
    "io.k8s.api.apps.v1beta1.Deployment": {
-    "description": "Deployment enables declarative updates for Pods and ReplicaSets.",
+    "description": "DEPRECATED - This group version of Deployment is deprecated by apps/v1beta2/Deployment. See the release notes for more information. Deployment enables declarative updates for Pods and ReplicaSets.",
```

Comment:
does the keyword `DEPRECATED` have a special significance ?

Reply:
Users consume it from API docs, e.g. https://kubernetes.io/docs/api-reference/v1.7/ or `kubectl explain`